src/components/Header/Hamburger.js

```diff
@@ -13,22 +13,24 @@ const HamburgerSvg = styled.svg`
 `
 
 const HamburgerButton = styled.button`
-  display: flex;
-  cursor: pointer;
-  margin: 0;
-  border: none;
-  background: transparent;
-  padding: ${remcalc(8)} ${remcalc(6)};
-
-  &:focus {
+  ${breakpoint('phone')`
+    display: flex;
+    cursor: pointer;
+    margin: 0;
+    border: none;
     background: transparent;
-    outline: ${remcalc(4)} solid ${props => props.theme.colors.outline};
-    color: ${props => props.theme.colors.text};
-  }
+    padding: ${remcalc(8)} ${remcalc(6)};
 
-  ${breakpoint('tablet')`
+    &:focus {
+      background: transparent;
+      outline: ${remcalc(4)} solid ${props => props.theme.colors.outline};
+      color: ${props => props.theme.colors.text};
+    }
+  `}
+
+  @media screen and (min-width: 960px) {
```

Comment:
now that we got our hands at it, we could try to use our `tablet` breakpoint (`901px`) instead of this 960... this way we could refer to it here instead of these 960px — which does not fit inside our breakpoint system. What do you think?

Reply:
It's actually intentionally `960px` in the design specs, rather than 'tablet' because this is the width taken up by that many links + their padding.

I'm rebasing onto master right now & the way I'm fixing it (temporarily) is by making this `1010px` instead of `960px` because it's only going to stay like this for as long as we have such a larger number of links on the TopNav.

As soon as we pick up [the new navigation structure](https://trello.com/c/4O2nPwzt/388-3-navigation-structure-on-topnav-desktop) (i.e. add the dropdowns into the top nav), this can go back to `960px`. So I'm doing that ticket next.

Reply:
👍